content/reference/endpoints/mid_term_rate_maps.md

```diff
@@ -17,7 +17,7 @@ Check out our guide: [Understanding Midterm Pricing](/guides/understanding-midte
   <div class="tab-pane active" id="rates_read" markdown="1">
 Name             | Type    | Read/Write | Description
 -----------------|---------|------------|------------
-id               | String  | Read       | Mid Term Rate Map's id.
+id               | BigInteger  | Read       | Mid Term Rate Map's id.
```

Comment:
@radekzawada but check the schema out, it's `id: :uuid`

Reply:
updated